bin/i18n/sync-out.rb

```diff
@@ -28,15 +28,21 @@ def sync_out
   I18nScriptUtils.run_standalone_script "dashboard/scripts/update_tts_i18n.rb"
 end
 
-# Files downloaded from Crowdin are organized by language name,
-# rename folders to be organized by locale
+# Files downloaded from Crowdin are organized by language name; rename folders
+# to be organized by locale, and remove any locales that are not in our system.
```

Comment:
>and remove any locales that are not in our system.
This function doesn't do this though, right?

Reply:
It does: the last line, specifically: https://github.com/code-dot-org/code-dot-org/pull/31771/files#diff-a11c4c1e91224504dd2ff6ee073b5047R45